burn-core/src/module/param/id.rs

```diff
@@ -6,7 +6,7 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Hash, PartialEq, Eq, Clone, Serialize, Deserialize)]
 pub struct ParamId {
-    value: String,
+    pub value: String,
```

Comment:
Can we make this crate private? I'm not sure we should leak the internal of how we store param ID, at some point we might use array of bytes of fixed length (`[u8; 16]`)

Reply:
OK. I'll make this private. I forgot to revert this. I initially needed this in my code serialization but not needed anymore.